mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -772,6 +772,53 @@ static FuncOp createCPUConvolution(ModuleOp &module, OpBuilder &builder,
   return cpuConvFuncOp;
 }
 
+static AllocOp getConvertedCpuResults(ModuleOp &module, OpBuilder &builder,
```

Comment:
The return value of this function is strange, we don't really care if it's an `AllocOp` or not. It's really a `Value`.

The same applies to other places where `AllocOp` is explicitly declared. We don't really use the result as an `AllocOp`, instead we treat them as a `Value`.